src/components/dashboard/InsightsAdvisorCard/InfoPopover.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+import { Icon, Popover } from '@patternfly/react-core';
+import { HelpIcon } from '@patternfly/react-icons/dist/esm/icons/help-icon';
+
+import { PLATFORM_LIGHTSPEED_REBRAND } from '~/queries/featureGates/featureConstants';
+import { useFeatureGate } from '~/queries/featureGates/useFetchFeatureGate';
+
+import links from '../../../common/installLinks.mjs';
+import ExternalLink from '../../common/ExternalLink';
+
+const InfoPopover = () => {
+  const allowPlatformLightspeedRebrand = useFeatureGate(PLATFORM_LIGHTSPEED_REBRAND);
+  const productName = allowPlatformLightspeedRebrand ? 'Red Hat Lightspeed' : 'Insights';
+
+  const informationText = [
+    `${productName} identifies and prioritizes risks to security, performance, availability, and stability of your clusters.`,
+    `This feature uses the Remote Health functionality of OpenShift Container Platform. For further details about ${productName}, see the `,
+  ];
+
+  return (
+    <Popover
+      className="openshift ocm-insights--info-popover"
+      aria-label={`What is ${productName}?`}
+      position="left"
+      maxWidth="25rem"
+      enableFlip
+      bodyContent={
+        <>
+          <p>{informationText[0]}</p>
+          <p>
+            {informationText[1]}
+            <ExternalLink href={links.REMOTE_HEALTH_INSIGHTS}>OpenShift documentation</ExternalLink>
+          </p>
+        </>
+      }
+    >
+      {/* <Icon className="ocm-insights--info-popover__icon" data-testId="recommendations-card-info-popover-icon"> */}
+      <Icon
+        className="ocm-insights--info-popover__icon"
+        role="img"
+        aria-label="recommendations-card-info-popover-icon"
```

Comment:
You are using the aria label as an id, this is a user-facing description of what the button does. Please update it to something meaningful like "Learn more on..." or similar.

Also the `role` looks wrong. This is a clickable item in the end that will trigger the pop-up. `button` should be right one.